src/features/images/DeleteImagesByFilterModal.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  deleteImagesByFilterTask,
+  fetchTask,
+  selectDeleteImagesByFilterLoading,
+} from '../tasks/tasksSlice.js';
+import { selectActiveFilters } from '../filters/filtersSlice.js';
+import { SimpleSpinner, SpinnerOverlay } from '../../components/Spinner.jsx';
+import { ButtonRow, HelperText } from '../../components/Form.jsx';
+import Button from '../../components/Button.jsx';
+import NoneFoundAlert from '../../components/NoneFoundAlert.jsx';
+import { deleteImages, selectImagesCount, selectImagesCountLoading } from './imagesSlice.js';
+import { setModalOpen, setModalContent } from '../projects/projectsSlice.js';
+import { IMAGE_QUERY_LIMITS } from '../../config';
+import { selectWorkingImages } from '../review/reviewSlice.js';
+
+const DeleteImagesByFilterModal = () => {
```

Comment:
@jue-henry do we need to use a modal at all for this? I feel like we may be able to get away with just using the `DeleteImagesAlert.jsx` component in a similar way to the pattern we use when it's opened from the [`LoupeDropdown.jsx`](https://github.com/tnc-ca-geo/animl-frontend/blob/main/src/features/loupe/LoupeDropdown.jsx).

There is some logic you implemented here that we'd definitely want to pull into the `DeleteImagesAlert.jsx` like the spinner, task polling, and the logic to determine which deletion action to dispatch, but I think ultimately that would keep things a littler simpler? I only really use modals when there's a decent amount of content to display; for something like this an alert might be sufficient. LMK if you feel otherwise though I'm happy to chat about it.

Reply:
I think your suggestion makes sense, implementing